### ISSUE TITLE
No warning about certificate expiration/changes

### ISSUE THREAD

**Issue by turt2live** [element-hq/element-web#6327]
### Description

My Let's Encrypt certificate expired because my cron job wasn't configured correctly (oops). When I opened riot-android, it yelled happily about a certificate change/expiration, however riot-web just sat there in a silent, sad, pile of being offline. 

### Version information

- **Platform**: web (in-browser)
- **Browser**: Chrome 64
- **OS**: Windows 10
- **URL**: riot.im/develop

**Comment by lampholder** [comment]
I tripped over this a bunch of times too (until I fixed my cron job) :P

**Comment by t3chguy** [comment]
But in browser I don't think the browser tells us this :(

**Comment by t3chguy** [comment]
Yeah this is not a thing the browser tells us, we just get a generic CORS failure
